### DISCUSSION TITLE
How to use on Debian-based Linux distros ?

### DISCUSSION THREAD

**Discussion by KaKi87** [Tyrrrz/DiscordChatExporter#1186] [Q&A] [answered]
Hello,

I'm trying to use the app on a Debian-based linux distro (e.g. Pop, Mint, Ubuntu).

The README says the CLI version supports Linux, but :

- there is no Linux executable in `DiscordChatExporter.Cli.zip` ;
- there is no explanation on how to use with Docker ;
- AUR and Nix aren't compatible with Debian.

What to do ?

Thanks

**Comment by Tyrrrz** [comment, marked as answer]
Docs

https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Linux.md

**Reply by KaKi87** [reply]
Thanks, it would be nice to directly link to these in the README.

**Reply by 96-LB** [reply]
![image](https://github.com/Tyrrrz/DiscordChatExporter/assets/21089343/6e2670d2-1e74-4d16-98df-75a364464082)

**Reply by KaKi87** [reply]
By *directly* I meant the specific page you linked me to here, and in an explicit way.

As an end user, the section I focused on is *Download*, so I was looking there for Linux installation instructions.

Thanks
